fastlane/docs/Actions.md

```diff
@@ -1085,16 +1085,76 @@ nexus_upload(
 
 ### [Appetize.io](https://appetize.io/)
 
-Upload your zipped app to Appetize.io
+Upload your zipped app to Appetize.io to stream your app in the browser. This requires your app to be built for the iOS simulator and not the device.
```

Comment:
Move `This requires your app to be built for the iOS simulator and not the device.` to later after the example so the narrative flow is from high-level to detailed.